starter-core/src/main/java/io/micronaut/starter/feature/function/gcp/template/gcpFunctionGroovyJunit.rocker.raw

```diff
@@ -16,10 +16,21 @@ import io.micronaut.gcp.function.http.*
 class @project.getClassName()FunctionTest {
 
     @@Test
-    void testFunction() {
+    void testGet() {
         new HttpFunction().withCloseable { HttpFunction function ->
-           GoogleHttpResponse response = function.invoke(HttpMethod.GET, "/@project.getPropertyName()")
+           GoogleHttpResponse response = function.invoke(HttpMethod.GET, "/")
             assert response.status == HttpStatus.OK
```

Comment:
We should probably rewrite this as : 

`assertEquals(HttpStatus.OK, response.status))`

Reply:
I've done this change.